### PR TITLE
upgrade storybook

### PR DESCRIPTION
**Pull request by ritz078**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5,34 +5,12 @@
 "@babel/code-frame@7.0.0", "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```
Comment (azizhk) [inline yarn.lock]: Upgrade your yarn version.
Don't delete integrity checks

File: yarn.lock [hunk 2]
```diff
@@ -10393,86 +9580,77 @@ react-test-renderer@^16.0.0-0, react-test-renderer@^16.7.0:
...
   version "16.7.0"
   resolved "https://registry.yarnpkg.com/react/-/react-16.7.0.tgz#b674ec396b0a5715873b350446f7ea0802ab6381"
-  integrity sha512-StCz3QY8lxTb5cl2HJxjwLFOXPIFQp+p+hxQfc8WE0QiLfCtIlKj8/+5tjjKm8uSTlAW+fCPaavGFS06V9Ar3A==
   dependencies:
     loose-envify "^1.1.0"
     object-assign "^4.1.1"
     prop-types "^15.6.2"
     scheduler "^0.12.0"
 
+react@^16.8.1:
```
Comment (azizhk) [inline yarn.lock]: Multiple versions of react. Might create problems.

**Comment by azizhk** [comment]
Even I was working on the same 😅
https://github.com/ritz078/pebble/compare/upgrade_storybook?expand=1

**Comment by ritz078** [comment]
continue on your PR. It has more updates. I am closing this.

**Pull request closed without merging**